roles/nginx_configs/meta/main.yml

```diff
@@ -3,5 +3,3 @@ dependencies:
   - { role: nginx }
   - { role: users }
   - { role: passenger }
-  - { role: php }
```

Comment:
Add `php` as a dependency for that other role?

I think what we want is to make a role for every app we have, so we can specify the dependencies per app.
This would be the playbook we want to get to:
```
# events.yml
name: Set-up events, events.ugent.be
hosts: events
user: root
roles:
    - gandalf
```